src/caja-file-management-properties.c

```diff
@@ -94,6 +94,12 @@ static const char * const zoom_values[] =
     NULL
 };
 
+/*
+ * This array corresponds to the object with id "model2" in
+ * caja-file-management-properties.ui. It has to positionally match with it.
+ * The purpose is to map values from a combo box to values of the gsettings
+ * enum.
+ */
```

Comment:
Thanks for adding this comment! It should reduce future bugs like that missing "By Size on Disk" :slightly_smiling_face:

Reply:
That was the intention. :slightly_smiling_face:

I thought about moving all those GSettings enum values to the GUI file as a column in the `GtkListStore`, which would avoid future errors. But I'm not sure it is the best way and also not so much related to this pull request.